sale_brand/models/sale_order.py

```diff
@@ -45,3 +46,11 @@ def default_get(self, fields_list):
             res["brand_id"] = team.brand_id.id
 
         return res
+
+    @api.model
+    def _default_note(self):
+        if self.brand_id:
+            if self.brand_id.terms_type == "html":
+                return _("Terms & Conditions: %s", self.brand_id.terms_url)
+            return self.brand_id.invoice_terms or ""
+        return super()._default_note()
```

Comment:
in which base module this method exist?

Reply:
the method exists in the sale module

Reply:
sorry I can't find it, can you give me a link,

Reply:
https://github.com/odoo/odoo/blob/de707dd3751c4d0fa21f7961057daffb6f437d93/addons/sale/models/sale_order.py#L111

Reply:
thnaks